getting_started/step_by_step/your_first_game.rst

```diff
@@ -740,7 +723,7 @@ instance.
 
     extends Node
 
-    export (PackedScene) var Mob
+    export(PackedScene) var _mob_scene
```

Comment:
This change violates the GDScript style guide:

> Also use PascalCase when loading a class into a constant or a variable

In addition, `export` is not a function - removing the space before the export type hint makes it seem so.

Reply:
Scenes are neither classes nor constants, and in all other instances of the demos exported PackedScenes are using snake_case so it would have to be changed there too.

Reply:
Well, we could discuss changing the style guide, but either way this code and the style guide should match.

Reply:
Actually, there is not a single example of `export(PackedScene)` in the GDScript style guide, so there's nothing that says this should be capitalized.

Reply:
@NathanLovato has this changed at some point? I believe it used to be included in this section:

>Also use PascalCase when loading a class into a constant or a variable:

> const Weapon = preload("res://weapon.gd")

There's no explicit PackedScene example in the guide now.

I'm fine either way, but this is a debate for the style guide, not for this PR. Whatever is preferred should be added there and followed.

Reply:
Another interesting thing is that without the `scene` suffix and without the `instance` suffix, we lose the ability to explicitly refer to everything in C#. `mob` would be the instance, `Mob` is the class, so whether we went with a `mob` field or a `Mob` property we would be masking something somewhere.